atlp-rwanda/atlp-devpulse-fn
# Clear Trash

## What does this PR do?

Clear Trash (Permanent delete of soft-deleted trainees)

## Description of Task to be completed?

1. Emptying Trash (Recyclebin)


## How should this be manually tested?

- `Clone the repo`
- checkout` to branch `ft-clear-recylebin` by running git checkout ft-clear-recylebin
- Install packages, run npm i or  run npm install
- Run server and app
- Go http://localhost:3000/trash, a list of soft deleted trainees will appear

![112234567](https://user-images.githubusercontent.com/86821764/198874618-1c9c8a2d-124a-4911-844a-8f17d2733910.PNG)
Click `Empty Trash` button and observer the results.
